src/qibolab/dummy.yml

```
@@ -59,24 +59,60 @@ native_gates:
                 type: qf}
             - {type: virtual_z, phase: 0.0, qubit: 0}
             - {type: virtual_z, phase: 0.0, qubit: 2}
+            - {duration: 30, amplitude: 0.05, shape: Rectangular(), coupler: 0, relative_start: 0,
+                type: coupler}
```

Maybe this needs to be 
```suggestion
                type: cf}
```
?
Actually it would be useful to have a test if these are parsed properly, ie check if `platform.pairs[(0, 2)].native_gates.CZ.sequence()` contains the proper pulses.